src/askui_runner/modules/core/infrastructure/runner/askui.py

```diff
@@ -74,20 +74,17 @@ def create_jinja_env(self, dir_path: str) -> jinja2.Environment:
 
     def render_templates(self, dir_path: str) -> None:
         jinja_env = self.create_jinja_env(dir_path=dir_path)
-        for root, _, files in os.walk(dir_path):
-            for file in files:
-                if not file.endswith(".jinja"):
-                    continue
-                template_file_path = os.path.relpath(os.path.join(root, file), dir_path)
-                target_file_path = os.path.join(
-                    root, file[:-6]
-                )  # TODO Use jinja literal for -6
-                with create_and_open(target_file_path, "w") as f:
-                    f.write(
-                        jinja_env.get_template(template_file_path).render(
-                            self.config.dict()
-                        )
+        templates = [template for template in jinja_env.list_templates() if template.endswith(".jinja")]
```

Comment:
```suggestion
        templates = jinja_env.list_templates(extensions=[TEMPLATE_EXTENSION])
```

Assuming the global constant `TEMPLATE_EXTENSION` is declared, set to `.jinja` and made usable inside the method...